src/fhirtypes/ValueSet.ts

```diff
@@ -83,6 +82,9 @@ export type ValueSetCompose = {
 
 export type ValueSetComposeIncludeOrExclude = {
   system?: string;
+  _system?: {
+    extension?: Extension[];
+  };
```

Comment:
I think `_system` is better represented as an [Element](https://github.com/FHIR/sushi/blob/contained-codesystem-in-valueset/src/fhirtypes/dataTypes.ts#L62-L65) since it technically could have an `id` or `extension` array.